src/modules/control.cpp

```diff
@@ -34,13 +34,70 @@ static uint16_t targetVolume;
 static uint32_t totalBreathTime;
 static uint32_t targetInhalationTime;
 
+// Added
+static uint32_t targetExhalationTime;
+
+// Added
+static uint16_t targetPressure;
+
 static uint32_t measuredInhalationTime;
 static uint32_t measuredExhalationTime;
 
+// Control variables
+static uint16_t motorCommandPosition;		// Commands to low-level controller
+static uint16_t motorCommandVelocity;
+
+static uint16_t desiredVolume;				// Where we set desired volume to reach
+static uint16_t desiredPressire;			// " " ...
+static uint16_t desiredFlow;
+static uint16_t currentVolume;				// Need to update volume measurement here from sensors
+static uint16_t currentPressure;			// " " ...
+static uint16_t currentFlow;
```

Comment:
`desiredVolume` is given as `targetVolume`, do we need two variables for the same thing or are they different? `currentXXX` should be taken from `sensors.currentXXX`

Reply:
It is different because the target volume represents the parameter (as in the case of volume control), where desired volume is the instantaneous set point. So in the exhalation phase, the desired volume would be 0 because we are heading backwards.